specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2022-01-01-preview/managedservices.json

```diff
@@ -1168,6 +1201,39 @@
           "type": "object"
         }
       }
+    },
+    "SystemData": {
```

Comment:
For SystemData, recommend to ref to common-type definition if possible：
Ref to specification\common-types\resource-management\v3\types.json

Reply:
> > e sure this PR is merge-ready (but don't merge yet)? We will give ack on merge as soon as the new API is live.
> 
> Hi @skayani, For registrationDefinitionIdParameter, as it is breaking your latest stable version, if your previous preview version is not get breaking change review board approval, you may have to get breaking change review board approval. if yes. can you show me the PR# for previous preivew version's approval? And for systemData, it is ok now, but do you have a roughly plan to support it?

HI @akning-ms , [here's the PR](https://github.com/Azure/azure-rest-api-specs/pull/8913) where we introduced the 2020-02-01-preview API. We didn't need breaking changes review. For the SystemData question, there aren't plans to support it now as the MarketplaceRegistrationDefinition resource isn't customer-accessible (it's an internally used resource). We may revisit this in the near future.

Reply:
Also @akning-ms , what exactly do I use for the $ref path value? I couldn't find any other RPs using that path

Reply:
Thanks Skayani, as the PR for 2020-02-01-preivew is merged before our cross API breaking change((mainly track the break for stable version)
) enable. Looks there is no any approval from breaking change review board.  so you may need to get breaking change review board approval, you can follow guide to do it with some justification andcurrent status of 2020-02-01-preview. e.g is there any SDK released with the version> . https://github.com/Azure/azure-rest-api-specs/pull/17089#issuecomment-1018250236

Reply:
For systemData, you can refer to this example:
https://github.com/Azure/azure-rest-api-specs/blob/e88a7008354fa35cdb3c551d6c7f9eb6b1e21db0/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2021-09-01/extensions.json#L691
systemData is same for v2 v3, just ensure use same version among your swagger

Reply:
Hey @akning-ms , we would rather avoid the breaking change review so I have reverted the offending breaking change. Please let me know when the PR is merge ready as I have made your suggestions

Reply:
Thanks skayani, After revert, breaking change review is not required, Once ARM sign off the PR, I will merge this PR.
If urgent:  please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them.

Reply:
@akning-ms Now that we have ARM review, are we good to go?